get_contact.py

```diff
@@ -0,0 +1,46 @@
+import base64
+import json
+import requests
+import sys
+
+def get_readme(owner, repo):
+    repo_name = owner + '/' + repo
+    url = 'https://api.github.com/repos/' + repo_name + '/readme'
+
+    r = requests.get(url)
+    j = json.loads(r.text)
+
+    return base64.decodestring(j[u'content'])
+
+def get_contact_for_user(owner):
+    url = 'https://api.github.com/users/' + owner
+
+    r = requests.get(url)
+    j = json.loads(r.text)
+
+    return j['email']
+
+def get_contact_for_org(owner, repo):
+    url = 'https://api.github.com/repos/' + owner + '/' + repo
```

Comment:
This will be used many times. In order to make only one `GET` request, this should be extracted to another file (perhaps `utils.py`?).